### PR TITLE
Moved some stuff

### PR DESCRIPTION
**Pull request by Rhydium**
- The code and styling that was in home.ejs have their own files now.

### CODE REVIEW

**Comment by tanwanimohit** [comment]
Hello, @Rhydium As i have merged some PRs, it is showing conflicts. so can you resolve it? if you can't just let me know i will do it !

**Comment by Rhydium** [comment]
Should be fixed.

**Comment by tanwanimohit** [comment]
> Should be fixed.

Still not fixed ! actually the problem is i recently merged some PR, so somebody has added Ace editor and you just removed that code.. you can do one thing clone it again and make your changes again.

OR 

add this updated code in code.js

` <script>
	let editor = ace.edit("content");
	editor.setTheme("ace/theme/eclipse");
	editor.setOption("fontSize", 17);
	editor.setOption("printMarginColumn", "60");
	editor.session.setMode("ace/mode/html");
		
	function make()
	{
		var content=editor.getValue();
		if(content!="")
		{
			$.post('/makehtml',{content:content},function(data,status){							
				if(data=="ERR1")
				{	
					document.getElementById("url").innerHTML="Ahh! you forget to write the code";
				}
				else
				{
					document.getElementById("url").innerHTML="Your Webpage is live!<br> <a style='font-color:dodgerblue; text-decoration:none;' href='"+data+"' >"+data+"</a>" ;
				}
			});
		}
		else
		{
			alert("Ahh! you forget to write the code");
		}
		
	}
	function load()
	{
		editor.setValue("<!DOCTYPE html>\r\n<html>\r\n<head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\r\n\r\n<title>My Web Page<\/title>\r\n<style>\r\n\r\n<\/style>\r\n<\/head>\r\n<body>\r\n\r\n\r\n<script>\r\n\r\n<\/script>\r\n<\/body>\r\n<\/html>\r\n");
	}
	function Copy()
	{
	
		var copyText = document.getElementById("content");
		copyText.select();
		document.execCommand("copy");
		document.getElementById("copybtn").value="Copied";
	}
	</script>`

**Comment by Rhydium** [comment]
I moved the code. Should be really fine now.

File: views/home.ejs [hunk 1]
```diff
@@ -11,20 +11,8 @@
     <link rel="stylesheet" href="assets/css/Highlight-Clean.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
     <link rel="stylesheet" href="assets/css/styles.css">
+    <script src="assets/js/code.js"></script>
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.0/jquery.min.js"></script>
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/ace/1.4.6/ace.js"></script>
```
Comment (tanwanimohit) [inline views/home.ejs:15]: This line is removed ! It should not be removed

File: views/home.ejs [hunk 2]
```diff
@@ -33,58 +21,14 @@
             <div class="intro">
                 <h1 class="text-center" style="font-family: 'Noto Sans JP', sans-serif; color:DodgerBlue; font-weight:bold;x "> &lt; &#47; &gt; Host.html </h1>
                 <p class="text-center">Host your HTML Files for Free, just put your content below and hit enter, you will get the unique URL use it to see your live web page.</p>
-			</div>
```
Comment (tanwanimohit) [inline views/home.ejs:36]: This line also Removed ! It should be there

**Review by tanwanimohit — COMMENTED** [review]
2 Lines are removed ! just keep them and your PR will be ready to be merged..
Thankyou

**Pull request closed without merging**